public/components/common/custom-search-bar/custom-search-bar.tsx

```diff
@@ -49,51 +50,57 @@ export const CustomSearchBar = ({ filtersValues, ...props }) => {
     }
 
 
-    const buildCustomFilter = (isPinned: boolean, index?: any, querySearch?: any, key?: any): Filter => {
+    const buildCustomFilter = (isPinned: boolean, values?: any): Filter => {
+        const newFilters = []
+        values.forEach(element => {
+            const filter = {
+                match_phrase: {
+                    [element.value]: element.label
+                }
+            }
+            newFilters.push(filter);
+        });
```

Comment:
**nitpick**: You could use `Array.prototype.map` instead of `Array.prototype.forEach`;

```ts
const newFilters = values.map(element => {
  const filter = {
    match_phrase: {
        [element.value]: element.label
    }
  }
});
```

The `match_phrase` give problem with some app logic. Use
```
match: {
  [element.value]: {
    query: element.value,
    type: 'phrase'
  }
}
```